recipes-wpe/wpe/wpe_0.1.bb

```diff
@@ -186,4 +186,7 @@ RDEPS_MEDIASOURCE += "${RDEPS_EXTRA}"
 RDEPS_VIDEO += "${RDEPS_EXTRA}"
 RDEPS_WEBAUDIO += "${RDEPS_EXTRA}"
 
-RRECOMMENDS_${PN} += "ca-certificates"
+RRECOMMENDS_${PN} += " \
+    ca-certificates \
+    ttf-bitstream-vera \
```

Comment:
Bitstream Vera has a very liberal license. That's one of the reasons for choosing it here as the safe default recommendation. The wpe-image in meta-metrological is Raspberry Pi specific, so it's not reasonable to expect all new users to use it.